nodeJS/express/installing_postgresql.md

```diff
@@ -123,16 +123,16 @@ You should see the PostgreSQL prompt come up with the new role we just created,
 role_name=#
 ```
 
-If you don't see a similar prompt, then reach out in [our Discord server](https://discord.gg/V75WSQG) for some help. If you **do** see a similar prompt, then we can create a password for the role like so:
+If you don't see a similar prompt, then reach out in [our Discord server](https://discord.gg/V75WSQG) for some help. If you **do** see a similar prompt, then we can create a password for the role like so (with a backward slash \):
```

Comment:
I feel like we don't really need to specify the backward slash given the full command is visible and it's not much of a "gotcha". Plus, `\)` won't render properly on our website by itself, since the `\` will be treated as an escape for `)`. Same applies across all instances of this addition.